reana_client/api/client.py

```diff
@@ -33,7 +33,10 @@
 
 
 def ping(access_token):
-    """Health check REANA server."""
+    """Health check REANA server.
```

Comment:
What do you think about rephrasing this to start the phrase with a verb, as for all the other docstrings?